packages/lesswrong/components/sunshineDashboard/UsersReviewInfoCard.tsx

```diff
@@ -0,0 +1,691 @@
+  downvotes: {
+    color: theme.palette.error.dark,
+    opacity: .75,
+    padding: 6,
+    paddingTop: 3,
+    paddingBottom: 3,
+    marginRight:8,
+    borderRadius: "50%",
+  },
+  upvotes: {
+    color: theme.palette.primary.dark,
+    opacity: .75,
+    padding: 6,
+    paddingTop: 3,
+    paddingBottom: 3,
+    marginRight:8,
+    borderRadius: "50%",
+  },
+  bigUpvotes: {
+    color: theme.palette.primary.dark,
+    padding: 6,
+    paddingTop: 3,
+    paddingBottom: 3,
+    marginRight:8,
+    borderRadius: "50%",
+    fontWeight: 600,
+  },
+  votesRow: {
+    marginTop: 12,
+    marginBottom: 12
+  },
+  notes: {
+    border: theme.palette.border.normal,
+    borderRadius: 2,
+    paddingLeft: 8,
+    paddingRight: 8,
+    paddingTop: 4,
+    paddingBottom: 4,
+    marginTop: 8,
+    marginBottom: 8,
+  },
+  sortButton: {
+    marginLeft: 6,
+    cursor: "pointer"
+  },
+  sortSelected: {
+    color: theme.palette.grey[900]
+  },
+  bio: {
+    '& a': {
+      color: theme.palette.primary.main,
+    }
+  },
+  website: {
+    color: theme.palette.primary.main,
+  },
+  info: {
+    // '& > * + *': {
+    //   marginTop: 8,
+    // },
+    display: 'flex',
+    flexWrap: 'wrap'
+  },
+  modButton:{
+    marginTop: 6,
+    marginRight: 16,
+    cursor: "pointer",
+    '&:hover': {
+      opacity: .5
+    }
+  },
+  snooze10: {
+    color: theme.palette.primary.main,
+    fontSize: 34,
+    marginTop: 4
+  },
+  permissionsButton: {
+    padding: 6,
+    paddingTop: 3,
+    paddingBottom: 3,
+    border: theme.palette.border.normal,
+    borderRadius: 2,
+    marginRight: 8,
+    marginBottom: 8,
+    cursor: "pointer",
+    whiteSpace: "nowrap"
+  },
+  permissionDisabled: {
+    border: "none"
+  },
+  columns: {
+    display: 'flex'
+  },
+  infoColumn: {
+    width: '30%',
+    padding: 16,
+    borderRight: theme.palette.border.extraFaint,
+  },
+  contentColumn: {
+    width: '35%',
+    padding: 16,
+    borderRight: theme.palette.border.extraFaint,
+    position: "relative"
+  },
+  actionsColumn: {
+    width: '35%',
+    padding: 16
+  },
+  content: {
+    marginTop: 16,
+    marginBottom: 8,
+    borderTop: theme.palette.border.extraFaint
+  },
+  expandButton: {
+    display: "flex",
+    justifyContent: "right",
+    color: theme.palette.grey[500]
+  },
+  contentCollapsed: {
+    maxHeight: 300,
+    overflow: "hidden"
+  },
+  contentSummaryRow: {
+    display: "flex",
+    flexWrap: "wrap"
+  }
+})
+
+interface UserContentCountPartial {
+  postCount?: number,
+  commentCount?: number
+}
+
+export function getCurrentContentCount(user: UserContentCountPartial) {
+  const postCount = user.postCount ?? 0
+  const commentCount = user.commentCount ?? 0
+  return postCount + commentCount
+}
+
+export function getNewSnoozeUntilContentCount(user: UserContentCountPartial, contentCount: number) {
+  return getCurrentContentCount(user) + contentCount
+}
+
+const UsersReviewInfoCard = ({ user, refetch, currentUser, classes }: {
```

Comment:
This is a large new component, and it feels like much the code was copied from elsewhere - is that necessary? Could we factor out some of this?